examples/nidmm/continuous-acquire-multiple-points.py

```diff
@@ -0,0 +1,186 @@
+def ThrowOnError (vi, error_code):
+    error_message_request = nidmm_types.ErrorMessageRequest(
+        vi = vi,
+        error_code = error_code
+    )
+    error_message_response = nidmm_client.ErrorMessage(error_message_request)
+    raise Exception (error_message_response.error_message)
+
+try:
+    # Open session to NI-DMM with options
+    print('\nInitializing session...')
+    init_with_options_response = nidmm_client.InitWithOptions(nidmm_types.InitWithOptionsRequest(
+        session_name = session_name,
+        resource_name = resource,
+        id_query = False,
+        option_string = options
+    ))
+    vi = init_with_options_response.vi
+    CheckForError(vi, init_with_options_response.status)
+    print(f'Session initialized with name {session_name} and id {vi.id}.\n')
+
+    # Configure measurement
+    print('Configuring Measurement...')
+    config_measurement_response = nidmm_client.ConfigureMeasurementDigits(nidmm_types.ConfigureMeasurementDigitsRequest(
+        vi = vi,
+        measurement_function = measurementType,
+        range = config_range,
+        resolution_digits = resolution
+    ))
+    CheckForError(vi, config_measurement_response.status)
+    print('Measurement Configured.\n')
+
+    # Configure a multipoint acquisition
+    print('Configuring Multipoint Acquisition...')
+    config_multipoint_response = nidmm_client.ConfigureMultiPoint(nidmm_types.ConfigureMultiPointRequest(
+        vi = vi,
+        trigger_count = 1,
+        sample_count = 0,
+        sample_trigger = nidmm_types.SampleTrigger.SAMPLE_TRIGGER_NIDMM_VAL_IMMEDIATE,
+        sample_interval = 0.0
+    ))
+    CheckForError(vi, config_multipoint_response.status)
+    print('Multipoint Acquisition Configured.\n')
+
+    # Initiate Acquisition
+    print('Initiating Acquisition...')
+    initiate_acquisition_response = nidmm_client.Initiate(nidmm_types.InitiateRequest(
+        vi = vi
+    ))
+    CheckForError(vi, initiate_acquisition_response.status)
+    print('Acquisition Initiated.\n')
+
+    # Set while loop control
+    stop_measurement = False
+
+    # initialize variables
+    num_measurements = 0
+    sum_measurements = 0
+
+    while(not stop_measurement):
+        pts_available = 0
+        # Check available data
+        read_status_response = nidmm_client.ReadStatus(nidmm_types.ReadStatusRequest(
+            vi = vi
+        ))
+        CheckForError(vi, read_status_response.status)
+        pts_available = read_status_response.acquisition_backlog
+
+        # if there are more than MAXPTSTOREAD measurements
+        # available, set pts_available to MAXPTSTOREAD in order to
+        # avoid reallocating the array for measurements
+        pts_available = min(pts_available, MAXPTSTOREAD)
+
+        if pts_available > 0:
+
+            # Fetch data
+            fetch_multipoints_response = nidmm_client.FetchMultiPoint(nidmm_types.FetchMultiPointRequest(
```

Comment:
Can you continuously plot the acquired points on a graph using matplot lib? It would be easy as this is python example. refer to scope example (https://github.com/ni/grpc-device/blob/main/examples/niscope/graph-measurement.py) 

This might slightly deviate from the c example, but would be evident for the user if they can visualize the acquired measurements. 
Then you can remove printing all the messages to the console too, as the plot will stop & return to console to print the number of measurements & avg value.

Reply:
added graph plotting and changed the file name